static/src/javascripts/projects/common/modules/navigation/sticky.js

```diff
@@ -437,7 +437,7 @@ define([
                 // Header is not slim yet
                 this.$els.header.removeClass('l-header--is-slim');
                 this.$els.header.css({
-                    position:  'static',
+                    position:  'relative',
```

Comment:
The issue was that the `z-index` wasn't being applied because it is only honoured for `relative` and `absolute` positioned elements.

I noticed there were other z-indexes set by the sticky nav along with position static. I don't know what the purpose of these is, as they will not be honoured. Any ideas @Calanthe @uplne?